modules/ding_wayf_dk/modules/ding_wayf_dk_login/ding_wayf_dk_login.module

```diff
@@ -0,0 +1,255 @@
+<?php
+/**
+ * @file
+ * Adds support for user login through WAYF dk services.
+ */
+
+/**
+ * Delimiter for extracting a part of a field.
+ */
+define('DING_WAYF_DK_LOGIN_FIELD_DELIMITER', ':');
+
+/**
+ * Implements hook_menu_alter().
+ *
+ * Allow users to logout of WAYF as they may be logged into WAYF but not the
+ * drupal site/library system.
+ */
+function ding_wayf_dk_login_menu_alter(&$items) {
+  $items['wayf/logout']['access callback'] = TRUE;
+}
+
+/**
+ * Implements hook_wayf_dk_login_create_user().
+ */
+function ding_wayf_dk_login_wayf_dk_login_create_user($attributes) {
```

Comment:
Why is this a part of the login module and not the create user module? As I see it the function deals with user creation and thus seems to fit within the scope of the create user module. It does not seem to be called elsewhere.